extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementMetadata.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.OffsetDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Schema(description = "Contract's agreement metadata")
+public class ContractAgreementMetadata {
+    @Schema(description = "Cancellation's date and time", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private OffsetDateTime cancellationDateTime;
```

Comment:
don't add a new API endpoint + object when you're adjusting a current one.

also no unnecessary nesting in API models unless it's really unreasonable.

Reply:
So you only want 4 extra fields, all related to the termination but not grouped as such?

Reply:
Fixed.